### PR TITLE
Asynchronous Task Queue-#528

### PR DESCRIPTION
**Pull request by abdullai-t**
Functionalities add:
- [x]  Create task
- [x] Edit task 
- [x] pause task
- [x] resume task
- [x] list all tasks
- [x] delete task

Screenshot 2022-03-31 at 9 20 48 PM
Screenshot 2022-03-31 at 9 22 58 PM

### CODE REVIEW

**Review by frimpongopoku — APPROVED** [review]
LGTM! 

However, can we make this whole "task" page a bit user friendly? 
Like on the "add task" we could write a two-liner that describes what "tasks" do in layman terms? 
Same for the form placeholders and labels.  

What if we changed `recurring details` to `How often would you like this task to run?`
And 

`Function name` to `Choose an automatic process that we should run when its time...` Dont exactly know what that `fuction name` dropdown does... so you @abdullai-t , or others could coin up a better statement. But you get the point ....?

**Review by BradHN1 — APPROVED** [review]
LGTM - Added one minor fix for case of no existing tasks
